### PR TITLE
Unity integration 0.0.3

### PR DESCRIPTION
**Pull request by Ysabz**
### Summary 
This PR adds new unity features to the webapp.
### Details
- added the ability to change skyboxes
- the argument exception bug is fixed
- the UI packages are refactored (material-ui is replaced by mui and all the components are updated respectively)
- the name of camera angles are added

### How to test?
_create, save , load, preview experiences.  
- change skybox
### Checks 
- [x] Tested Changes
- [ ] Stakeholder Approval

### CODE REVIEW

File: pages/_document.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ServerStyleSheets as ServerStyleSheetsMUI } from "@material-ui/core";
+import { ServerStyleSheets as ServerStyleSheetsMUI } from "@mui/styles";
```
Comment (mckenzietsean) [inline pages/_document.tsx:1]: Site fails to compile. 
Module not found: Can't resolve '@mui/styles'

**Review by mckenzietsean — CHANGES_REQUESTED** [review]

File: pages/_document.tsx (hunk 1 above)
Comment (Ysabz) [reply pages/_document.tsx:1]: try yarn install. if that doesn't work, delete node module and package.lock and then try yarn install again

Comment (mckenzietsean) [reply pages/_document.tsx:1]: Never mind this, it was fixed with a "yarn install".
The rest of the PR looks good, however this request for change will still remain as there's still 1 code smell that needs to be fixed.

**Comment by shushengfuqin** [comment]
I was testing each apparatus added
The skybox looks great
For the bird cage, the close up works fine, however the action reset and active do not have modification
For the earth, nothing shows up and the actions aren't working neither
For the multi-asset, nothing shows up.

**Comment by Ysabz** [comment]
> I was testing each apparatus added The skybox looks great For the bird cage, the close up works fine, however the action reset and active do not have modification For the earth, nothing shows up and the actions aren't working neither For the multi-asset, nothing shows up.

The earth and multi asset are defective apparatuses. I removed them from container for now

**Comment by shushengfuqin** [comment]
> > I was testing each apparatus added The skybox looks great For the bird cage, the close up works fine, however the action reset and active do not have modification For the earth, nothing shows up and the actions aren't working neither For the multi-asset, nothing shows up.
> 
> The earth and multi asset are defective apparatuses. I removed them from container for now

Then I have no problem. 
If the main purpose of this pr is the skybox, they are working perfectly fine. The only thing is the time to change it is a bit long. Everything else looks wonderful.

**Review by shushengfuqin — APPROVED** [review]
lgtm

**Pull request merged**